nixos/modules/misc/locate.nix

```
@@ -76,7 +76,7 @@ in {
 
     prunePaths = mkOption {
       type = listOf path;
-      default = ["/tmp" "/var/tmp" "/var/cache" "/var/lock" "/var/run" "/var/spool" "/nix/store"];
+      default = ["/tmp" "/run" "/var/tmp" "/var/cache" "/var/lock" "/var/run" "/var/spool" "/nix/store"];
```

This probably should be
```suggestion
      default = ["/tmp" "/var/tmp" "/var/cache" "/run/lock" "/run" "/var/spool" "/nix/store"];
```

---

Ah, `lock` is part of `run`. Looked it up, yes indeed.